src/DSharp.Compiler/Importer/MetadataImporter.cs

```diff
@@ -537,6 +537,11 @@ private void ImportMethods(TypeSymbol typeSymbol)
                     returnType,
                     MetadataHelpers.IsExtensionMethod(method));
 
+                if(MetadataHelpers.ShouldIgnoreMethodGeneratedTypeArguments(method))
+                {
+                    methodSymbol.IgnoreGeneratedTypeArguments = true;
```

Comment:
assign the value directly from MetadataHelpers.ShouldIgnoreMethodGeneratedTypeArguments(method))